client/main.lua

```diff
@@ -26,6 +32,35 @@ RegisterNetEvent('qb-printer:client:SpawnPrinter', function()
     SetEntityAsMissionEntity(obj)
 end)
 
+RegisterNetEvent("qb-printer:use")
+AddEventHandler('qb-printer:use', function()
+    SendNUIMessage({
+        action = "start"
+    })
+    SetNuiFocus(true, true)
+end)
+
+-- Threads
+CreateThread(function()
+    while true do
+        local ped = PlayerPedId()
+        local pos = GetEntityCoords(ped)
+        for k, v in pairs(Config.PrinterModels) do
+            local PrinterObject = GetClosestObjectOfType(pos.x, pos.y, pos.z, 1.5, GetHashKey(v.model), false, false, false)
+            if PrinterObject ~= 0 then
+                local PrinterCoords = GetEntityCoords(PrinterObject)
+                exports['qb-target']:AddBoxZone("Printer", PrinterCoords, 0.4, 0.4, { name="Printer", heading = 0, debugPoly=false, minZ=PrinterCoords.z, maxZ=PrinterCoords.z+0.4 }, 
```

Comment:
```suggestion
                exports['qb-target']:AddBoxZone("Printer", PrinterCoords, 0.4, 0.4, { name="Printer", heading = 0, debugPoly=false, minZ=PrinterCoords.z, maxZ=PrinterCoords.z+0.4 },
```
Remove trailing whitepsace.